sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -1566,11 +1566,23 @@ private ServiceBusSessionProcessorClientBuilder() {
          * Sets the amount of time to continue auto-renewing the lock. Setting {@link Duration#ZERO} or {@code null}
          * disables auto-renewal. For {@link ServiceBusReceiveMode#RECEIVE_AND_DELETE RECEIVE_AND_DELETE} mode,
          * auto-renewal is disabled.
+         * <p>
+         * A Service Bus queue or subscription in a topic will have a lock duration set in the resource level.
+         * When the processor client connect to a session in the resource, the broker will apply an initial
+         * lock to the session. This initial lock only lasts for the lock duration set in the resource level.
+         * If the client does not renew the initial lock before it expires then the session will be released and become
+         * available for other receivers. Each time the client renews the lock, the broker will extend the lock for the
+         * lock duration set in the resource level. It means, to keep the session locked for a long time, the client may
```

Comment:
I think "at the resource level" is better than "in the resource level"?  but that might be me.

Reply:
agree, reworded